### DISCUSSION TITLE
Project deployment error

### DISCUSSION THREAD

**Discussion by DocZeus** [vercel/community#7125] [Help] [answered]
### Summary

Project name
Project names can be up to 100 characters long and must be lowercase. They can include letters, digits, and the following characters: '.', '_', '-'. However, they cannot contain the sequence '---'.
I have no such sequence but I'm still getting this error. I've checked my project names both in github and here while deploying. Also check each one of my files(except node_modules since that's getting built anyway) and none of them contain the sequence mentioned in the error.
![vercel](https://github.com/vercel/community/assets/66434877/ceee527c-1e1d-4e72-a896-56f62f5b0f0c)

### Steps to Reproduce

Try to deploy any code from your github profile in vercel. Error should prompt when you click on the deployment button.

**Comment by WingSky2022** [comment]
Have you solved this problem?

**Reply by DocZeus** [reply, marked as answer]
> Have you solved this problem?

yes, this was a reading mistake on my part. If you notice what the instructions say, "must be lowercase", so inserting the name "Notes" was invalid. "notes" is acceptable.

[a comment by an automated account is not shown]
